doc/classes/MarginContainer.xml

```diff
@@ -4,22 +4,30 @@
 		Simple margin container.
 	</brief_description>
 	<description>
-		Simple margin container. Adds a left margin to anything contained.
+		Simple margin container. Adds a top,left,bottom,right margins to anything contained.
```

Comment:
Suggestion:

`Adds a top, left, bottom, and right margin to all [Control] nodes that are direct children of the container.
To control the [MarginContainer]'s margin, use the [code]margin_*[/code] theme properties listed below.`